lib/tasks/bounty_tasks.rake

```diff
@@ -0,0 +1,10 @@
+desc 'send open bounties email'
+task send_bounty_email: :environment do
+  @bounties = Bounty.where(status: "open").last(15)
+  if @bounties.any?
+    @users = User.all
+    @users.each do |user|
+      BountyMailer.with(user: user, bounties: @bounties).open_bounty.deliver_later
+    end
+  end
+end
```

Comment:
I mentioned above how Rails uses a param hash for the data we pass the our mailers, that's what's happening here.  We get all the open bounties and set to `@bounties` if there are 0 bounties, we shouldn't do anything.

If there _are_ bounties, we loop through each one of our Users and send them the BountyMailer with the current  bounties.